index.html

```diff
@@ -0,0 +1,187 @@
+        <h2 class="headline-heading2">
+          BECOME A PROFESSIONAL REMOTE WEB DEVELOPER
+        </h2>
+        <p class="headline-details">
+          Join the community of world class web developer around the world, and
+          become a professional remote web developer. Change your career into
+          tech now
+        </p>
+        <div class="headline-img-container">
+          <div>
+        <h2 class="headline-heading3">NEXT COHORT STARTING IN 15 DAYS</h2>
+        <p class="headline-details2">Book your enrolment now</p>
+        </div>
+        <img class="hero-img" alt="hero image" src="images/hero-bg.png">
+        </div>
+      </section>
+
+      <!-- #### highlight-content-sectionn #### -->
+      <section class="highlight-section" id="program">
+        <h2 class="highlight-heading">Why THECODECHASER</h2>
+        <hr>
+        <div class="hightlight-content">
+          <div class="highlight-item">
+            <img alt="highlight icon" class="highlight-item-icon" src="images/highlight-icon1.png">
+            <h3 class="highlight-item-heading">Projects</h3>
+            <p class="highlight-item-details">We believe in project based learning, you build industry level projects.</p>
+          </div>
+          <div class="highlight-item">
+            <img alt="highlight icon" class="highlight-item-icon" src="images/highlight-icon2.png">
+            <h3 class="highlight-item-heading">Material</h3>
+            <p class="highlight-item-details">We have best open-source study material from internet, That will teach you.</p>
+          </div>
+          <div class="highlight-item">
+            <img alt="highlight icon" class="highlight-item-icon" src="images/highlight-icon3.png">
+            <h3 class="highlight-item-heading">Patern</h3>
+            <p class="highlight-item-details">Our study pattern is diffrent from many online school, we focus on mastry learning.</p>
+          </div>
+          <div class="highlight-item">
+            <img alt="highlight icon" class="highlight-item-icon" src="images/highlight-icon4.png">
+            <h3 class="highlight-item-heading">Jobs</h3>
+            <p class="highlight-item-details">Our career coaches are help you to find your dream job, Get high paying jobs.</p>
+          </div>
+          <div class="highlight-item">
+            <img alt="highlight icon" class="highlight-item-icon" src="images/highlight-icon5.png">
+            <h3 class="highlight-item-heading">Help</h3>
+            <p class="highlight-item-details">Our staff is available for you all the time to help you with quastions.</p>
+          </div>
+        </div>
+        <button type="button" class="button join-us">Enroll now and confirm your spot</button>
+        <a class="highight-end" href="#">SEE THE WHOLE PROGRAM</a>
+      </section>
+
+      <!-- #### featured-courses-section #### -->
+      <section class="featured-tutors" id="tutors">
+      </section>
```

Comment:
- [ ] Please make sure to horizontally center this `section` on `desktop` screens to match the design.